packages/gguf/src/gguf.ts

```diff
@@ -56,6 +56,9 @@ class RangeView {
 	get view(): DataView {
 		return this.dataView;
 	}
+	get currentChunk(): number {
```

Comment:
looks like this method is only used inside `RangeViewLocalFile`. Therefore, you can move this method to  `RangeViewLocalFile` only

Reply:
Yeah I forgot that I can use `protected` keyword in ts. I was doing this way because `private chunk` can't be read by subclass.

It is fixed with https://github.com/huggingface/huggingface.js/pull/656/commits/e68fc5075acd75da9a7fb0e143e63faee9f82ff7